### PR TITLE
Feature demo sprint2

### PR DESCRIPTION
**Pull request by nielsopdam**
Small fixes in front end of bookings.
Added minty.css file

### CODE REVIEW

File: src/main/resources/public/js/bookings.js [hunk 1]
```diff
@@ -6,24 +6,47 @@ $(document).ready(function () {
...
+        return null;
+    }
+}
+
 function getBookings() {
     console.log("Getting All Bookings...");
 
     $.ajax({
-        url:"http://localhost:8080/api/booking/",
+        url:"/api/booking/",
```
Comment (jimmy1993k) [inline src/main/resources/public/js/bookings.js]: /api/bookings/

File: src/main/resources/public/js/bookings.js [hunk 2]
```diff
@@ -6,24 +6,47 @@ $(document).ready(function () {
...
         type:"get",
         success: function(bookings) {
             console.log("This is the data: " + bookings);
```
Comment (jimmy1993k) [inline src/main/resources/public/js/bookings.js]: console.log kan weg

**Review by jimmy1993k — COMMENTED** [review]
check comments

File: src/main/resources/public/js/bookings.js [hunk 3]
```diff
@@ -6,24 +6,47 @@ $(document).ready(function () {
...
-            { "data": "endDate" },
-            { "data": "checkedIn" },
-            { "data": "checkedOut" },
-            { "data": "paid" },
-            { "data": "paymentMethod" },
-            { "data": "duration" }
+            { "data": "rooms.0.id" },
+            { "data": "guests.0.firstName" },
+            { "data": "guests.0.lastName" },
+            { "mData": function date(data, type, dataToSet) {
```
Comment (AvantHof) [inline src/main/resources/public/js/bookings.js]: wat is mData?

File: src/main/resources/public/js/bookings.js [hunk 4]
```diff
@@ -6,24 +6,47 @@ $(document).ready(function () {
...
+                    return formatDate(data.startDate);
+                }
+            },
+            { "mData": function date(data, type, dataToSet) {
+                    return formatDate(data.endDate);
+                }
+            },
+            { "mData": function date(data, type, dataToSet) {
```
Comment (AvantHof) [inline src/main/resources/public/js/bookings.js:21]: Tijdstip ipv hele datum

File: src/main/resources/public/js/bookings.js [hunk 5]
```diff
@@ -6,24 +6,47 @@ $(document).ready(function () {
...
+function formatDate(date){
+    if(date){
+        return date.dayOfMonth + "-" + date.monthValue + "-" + date.year;
+    } else {
+        return null;
+    }
+}
+
 function getBookings() {
     console.log("Getting All Bookings...");
```
Comment (AvantHof) [inline src/main/resources/public/js/bookings.js:47]: console.log kan weg

**Review by AvantHof — CHANGES_REQUESTED** [review]

**Review by AvantHof — APPROVED** [review]

**Pull request merged**